### PR TITLE
[vikunja] Added link for postgresql and updated readme to support

### PR DESCRIPTION
**Pull request by carpenike**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/vikunja/Chart.yaml [hunk 1]
```diff
@@ -24,3 +24,7 @@ dependencies:
   - name: common
     repository: https://library-charts.k8s-at-home.com
     version: 3.3.0
+  - name: postgresql
+    version: 10.5.3
+    repository: https://charts.bitnami.com/bitnami
+    condition: postgresql.enabled
```
Comment (bjw-s) [inline charts/stable/vikunja/Chart.yaml:30]: Missing newline 👿

**Review by bjw-s — DISMISSED** [review]

File: charts/stable/vikunja/Chart.yaml (hunk 1 above)
Comment (carpenike) [reply charts/stable/vikunja/Chart.yaml]: done :)

**Comment by bjw-s** [comment]
Approved them before I noticed the pre-commit failed... Changes look good otherwise, so will approve once CI passes

**Comment by carpenike** [comment]
Yup noticed the trailing whitespace, should be cleaned.

**Review by bjw-s — APPROVED** [review]

**Pull request merged**